packages/design-system/src/components/patterns/thirdPartyExternalLinks.stories.tsx

```diff
@@ -13,11 +13,14 @@ export const ThirdPartyExternalLink = () => {
   return (
     <>
       <Button
-        className="ds-c-button ds-c-button--ghost ds-u-font-weight--normal"
+        className="ds-c-button--ghost ds-u-font-weight--normal"
         onClick={() => setShowModal({ showModal: true })}
       >
         Link to external site
-        <ExternalLinkIcon className="ds-u-margin-left--05" />
+        <ExternalLinkIcon
+          title="This link goes to an external site"
```

Comment:
The ticket had specific verbiage on what the `title` text should say, so I've used that exactly here, overwriting the default title set in the locale files. 

The locale files just say "External link" (in English and Spanish) and I've left that intact. Is that default title text enough context to inform users of that icon's meaning? Or should I update our locale files with this new content? If an update is recommended, I need a Spanish speaking resource to confirm a Google translate version is correct.